src/libs/antares/study/area/sets.cpp

```diff
@@ -239,17 +209,17 @@ bool Sets<T>::loadFromFile(const std::filesystem::path& filename)
 
                 if (p->key == "+")
                 {
-                    opts.rules.push_back(Rule(ruleAdd, new String(value)));
+                    opts.rules.push_back(Rule(ruleAdd, value.to<std::string>()));
```

Comment:
This is so much better, the previous code was very confusing regarding memory ownership